docs/src/style/index.md

```diff
@@ -156,13 +156,22 @@ This can be through inheriting from it or by directly copying artifacts.
 The main point is that a subproject should not have multiple builds scattered in various places.
 Each subproject has an authoritative `build` target that *all* targets use when fetching build artifacts.
 
-### Prefer User Defined Commands (UDCs)
-
-The primary purpose of a UDC is to reduce boilerplate and promote reusing common workflows.
-Many build patterns tend to be repetitive.
-For example, copying a package lockfile and installing dependencies is very common.
-
-In these cases, a UDC should be preferred.
-The `catalyst-ci` repository provides a number of UDCs in the `earthly` subdirectory.
-These should be used prior to writing a new one.
-If a common use case is not covered in this subdirectory, a PR should be opened to add it.
+### Use Functions instead as User Defined Commands (UDCs) is Deprecated in Earthly 0.8
```

Comment:
I think the original one is good already just change the word UDC to Function, since the functionality is the same, they just changed the word. 

Maybe add a warning section mention about the deprecation of UDC

Reply:
will read up on that bit and add a comprehensive but concise warning

Reply:
added the warning